misc-scripts/xref_mapping/XrefParser/VGNCParser.pm

```diff
@@ -15,6 +15,63 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
```

Comment:
This should be replaced with the note to look at the notice file. I missed this when merging #421 (ZFINDesc) - apologies